docs/source/markdown/options/device-write-iops.md

```diff
@@ -0,0 +1,5 @@
+#### **--device-write-iops**=*path*
+
+Limit write rate (IO per second) to a device (e.g. --device-write-iops=/dev/sda:1000)
```

Comment:
Please choose the version from `podman-run`, i.e., the one with 'path:rate' in the '####' line and asterisks in the example.